upgrade-guides/v4.0-v5.0.md

```diff
@@ -0,0 +1,37 @@
+# How to Upgrade Hazelcast-wm Version 4.0 to Version 5.0
+
+This guides summarizes what you need to know if you are upgrading from Hazelcast-wm 4.0 version to 5.0 version.
+
+
+## Hazelcast-wm supports Hazelcast 5.x
+
+Hazelcast-wm supported Hazelcast 4.x versions, whereas Hazelcast-wm supports Hazelcast 5.x. One notable change in Hazelcast 5.x 
+that can affect Hazelcast-wm usage is that Hazelcast clients have indefinite cluster connect timeout by default,
+which was previously 20 seconds. For more info on the topic, see
+[this page](https://docs.hazelcast.com/hazelcast/5.4/clients/java#configuring-client-connection-retry)
+
+To preserve the 20 seconds cluster timeout behaviour in Hazelcast 5.x clients, you need to add this configuration to your 
+hazelcast-client.xml config:
+
+```xml
+<connection-strategy>
+        <connection-retry>
+            <cluster-connect-timeout-millis>20000</cluster-connect-timeout-millis>
+        </connection-retry>
+    </connection-strategy>
+```
+
+## Dependencies are upgraded
```

Comment:
```suggestion
## Updated dependencies
```

Reply:
Are these dependencies or supported versions?

Reply:
changed to Supported versions of dependencies